corehq/apps/app_manager/util.py

```diff
@@ -409,7 +409,7 @@ def get_casedb_schema(form):
     This lists all case types and their properties for the given app.
     """
     app = form.get_app()
-    base_case_type = form.get_module().case_type
+    base_case_type = form.get_module().case_type if form.requires_case() else None
```

Comment:
This looks like a good change, but wondering if it will hide case properties from people who were previously seeing/using them?

Also, I'm wondering if this will fix the issue or if it would be better to roll back #15798 and give that a second try that doesn't have so much impact on performance? @dannyroberts may want to chime in on that.